### DISCUSSION TITLE
Beat Twitter

### DISCUSSION THREAD

**Discussion by sapien-systems** [mastodon/mastodon#24606] [General]
I'm Dr. Dan
https://sfba.social/@drdan
https://physixq2.com/

This is the innovation:
integrate the different feedback systems (thumbs up, 5-star, 1-10, etc.) into a single, language independent metric.

🔴= negative
🟢 = neutral
🔵 = positive
hate it, neutral, love it, and everything in between.
🏳️‍🌈

Additionally, using the grayscale, you can estimate crediblity -- how "true" something is.

You can now measure Quality and Credibility.  Exactly what Twitter lacks.

If Mastodon integrates this before Musk, it would be a game changer.

Who would I talk to about this?

**Comment by fishcharlie** [comment]
Just because something gets "negative" feedback, doesn't mean it's not credible or "true". There is a reason YouTube removed showing dislikes on videos publicly. There is a huge difference between "quality" & "credibility". Anyone with time and effort has the ability to make quality content, that doesn't make it credible. Additionally, what is the benefit of `neutral`? Why should I stop scrolling in my feed to mark something as `neutral` vs continuing to scroll? And on top of it, if something is truly `negative` or untrue, what makes this better than actually replying to the content in question and presenting the accurate facts with sources and research to back it up?

And on top of it, do I even need to mention that `neutral` being green makes zero sense?

In summary, this isn't a game changer. In my opinion this shouldn't even be implemented. Not thought through, not well designed. Doesn't solve a clear need. Too many questions. Just not necessary, and not a clear obvious benefit.

**Reply by sapien-systems** [reply]
You misunderstand: Color is preference, Grayscale is credibility.
It can be True and Bad or True and Good, False and Bad or False and Good.

Color = Qualia
You need a unit that works across language barriers that means the same.
A continuum makes sense, green is in the middle.

**Reply by fishcharlie** [reply]
@sapien-systems Yeah ok... you still barely addressed any of the points I made. I'd guess that you aren't getting much response here because this isn't a well thought out plan/idea. This isn't a game changer.

**Reply by sapien-systems** [reply]
> @sapien-systems Yeah ok... you still barely addressed any of the points I made.
YOU: Just because something gets "negative" feedback, doesn't mean it's not credible or "true"
--------------
You are 100% correct, that's why there are 2 metrics: subjective (positive/negative) and credibility (true/false).
that's how you identify and separate the natural instinct to disbelieve what you don't want to be true.

YOU:  There is a huge difference between "quality" & "credibility"
--------------
Precisely why there are 2 metrics.

YOU: what is the benefit of neutral? Why should I stop scrolling...
--------------
Good point, green is default, colors are only to be used if the content "moves the needle" + or -
Where it's important is when the system is used for consensus.
You need to know when someone moves from redshift to green, that means they are satisfied that the idea is benign even if they don't support it.

YOU: what makes this better than actually replying to the content in question
--------------
Replying to the content is the status quo: endless nonsense arguments and endless sensible arguments that are ignored.  This pattern is pervasive regardless of language.  Using colors allows cross-lingual comparison of response.  A productive argument is where someone opposed moves to green or blueshift.  Those color shifts mean that someone's mind was changed.  If it makes sense to one person, that same argument is likely to persuade many. The advantage is that the collected persuasive arguments have the same pattern regardless of language, exactly what AI can't do now.  It scales globally.

YOU: I'd guess that you aren't getting much response here because this isn't a well thought out plan/idea. This isn't a game changer.
--------------
There's a lot under the hood that you don't see.
I've got teachers waiting in the USA and Australia, once this gets a little traction, it will go global, there are a million apps that will add the feature.  I'd like to talk to someone at Mastodon before it hits Twitter.

If you are curious how it works for schools:
https://youtu.be/GOVVt6oxs4Q
https://physixq2.com/

@fishcharlie , are you a developer at Mastodon?
